src/components/ReviewsList.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import ReviewCard from "./ReviewCard";
+import { getAllReviews } from "../utils.js";
+
+export default function ReviewsList() {
+  const [reviews, setReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getAllReviews().then(({ reviews }) => {
+      setReviews(reviews);
+      setIsLoading(false);
+      console.log("in the useEffect");
+    });
+  }, []);
+
+  if (isLoading)
+    return (
+      <section className="loading">
+        ...loading...
```

Comment:
Great to see you have included a loading state for the user as a visual indicator 🥳